docs/en/02_Developer_Guides/01_Templates/02_Common_Variables.md

```diff
@@ -145,6 +145,27 @@ $MetaTags(false)
 <title>$Title - Bob's Fantasy Football</title>
 ```
 
+### Disabling the meta generator tag
+
+The meta generator tag includes the current version of `silverstripe/framework`. This version number
+provides aggregate installation numbers to the product team who maintain Silverstripe CMS which is
+used to make informed product decisions.
+
+If it's felt this creates a security concern for your website, then the version portion of the meta
+generator tag can be disabled via yaml configuration:
+
+```yml
+SilverStripe\CMS\Model\SiteTree:
+  show_meta_generator_version: false
+```
+
+The entire meta generator tag can be configured off via:
```

Comment:
For consistency and ease of understanding it may be better to use the same "disabled" language as above.
```suggestion
The entire meta generator tag can be disabled via:
```

Reply:
Updated